### ISSUE TITLE
auro-accordion: focus state has low color contrast (a11y)

### ISSUE THREAD

**Issue by geoffrich** [AlaskaAirlines/auro-accordion#9]
## Describe the bug
When focused, the accordion's text color uses the `auro-color-brand-breeze-300` token (#00cff0). This has a [contrast ratio of 1.87](https://contrast-ratio.com/#%2300cff0-on-white) on a white background which does not meet WCAG AA color contrast requirements of 4.5 : 1.

## To Reproduce
1. Focus the accordion
2. Check the color contrast

## Expected behavior
The color contrast meets WCAG AA requirements.

## Screenshots
![image](https://user-images.githubusercontent.com/4992896/106020267-6d3d2d80-6078-11eb-924c-9003d1e350f3.png)

**Comment by blackfalcon** [comment]
The issue at hand is that the accent color used was never intended to be used on light backgrounds. But going to the standard UI blue, that is pretty harsh. 

Took at stab at an approach here to darken the colors within the family and add a fill as we are commonly doing with most tab state UIs. 

https://auro-accordion.surge.sh/components/auro/accordion

Will consult with the design team.

**Comment by blackfalcon** [comment]
@vidalmenAS any support on this would be appreciated.
